app/application/dtos/analytics.py

```
@@ -0,0 +1,21 @@
+
+
+@dataclass
+class DashboardStats:
+    """Dashboard stats for a tenant: counts and recent activity."""
+
+    total_subjects: int
+    subjects_by_type: dict[str, int]
+    total_events: int
+    events_by_type: dict[str, int]
+    total_documents: int
+    recent_events: list["EventResult"]
```

Ruff (0.15.0)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)